### DISCUSSION TITLE
a tag style issues

### DISCUSSION THREAD

**Discussion by mydracula** [nuxt/nuxt#17182] [Questions]
### Environment

Operating System: Windows_10_LTSC
Node Version: v16.7.0
Nuxt Version: 3.0.0-rc.6
Package Manager: npm@8.1.1
Builder: vite
User Config: -
Runtime Modules: -
Build Modules: -

### Reproduction

review: https://stackblitz.com/edit/github-wagzme?file=app.vue

![image](https://user-images.githubusercontent.com/71320199/181678562-75306355-95b6-4146-b9e4-d104a27fbe1c.png)

### Describe the bug

Using the minimal changes created by the document, I found that the A tag does not have a default spacing when I did not use any styles, even with "display: inline-block" set, now I need this default spacing what do I do?

### Additional context

none

### Logs

```shell
none
```

**Comment by danielroe** [comment]
I'm not sure what the issue is. Would you clarify?

**Comment by mydracula** [comment]
A tag is an element in the line

![image](https://user-images.githubusercontent.com/71320199/181712102-df012d09-0e24-4169-a7b1-ddd2e3f50272.png)

Why does the effect of NUXT rendered and the effect of static HTML above the above performance is different

**Comment by danielroe** [comment]
I still don't understand what you are asking. But doesn't sound to me like a Nuxt issue. Is it a CSS question?

**Comment by Flanker72** [comment]
You may have a look on https://github.com/nuxt/framework/discussions/5961 - the same issue was resolved. You just need to pass `whitespace: preserve` config parameter to vue compiler.
